### PR TITLE
Show actual split to manager

### PR DESCRIPTION
**Pull request by siapankina**
story: https://www.pivotaltracker.com/story/show/180262836

If job was split by operative, manager will be able to see actual split results before closing the job for the operative

### CODE REVIEW

**Review by pixeltrix — APPROVED** [review]
Just a couple of queries but other than that 👍🏻

File: cypress/integration/work-order/update.spec.js [hunk 1]
```diff
@@ -1014,6 +1015,7 @@ describe('Updating a work order', () => {
           ],
           comments:
             'Work order updated - Assigned operatives Operative A : 30%, Operative B : 20%, Operative C : 50%',
+          is_split: true,
```
[the remark this answers is not in the document]
Comment (siapankina) [reply cypress/integration/work-order/update.spec.js:1018]: f/e suggested isSplitted , but b/e went with is_split 
I will ask to change it to camelcase to be consistent , probably will need to be done with separate PR if that's ok?

[the remark this answers is not in the document]
Comment (pixeltrix) [reply cypress/integration/work-order/update.spec.js:1018]: Yes, that's fine

File: src/components/WorkOrders/SelectPercentage.js [hunk 2]
```diff
@@ -102,7 +109,16 @@ const SelectPercentage = ({
...
     let some = preSelectedPercentages && selectedOperatives.length === 1
-    if (
+    if (showSplitByOperative && selectedOperativePercentage) {
+      setSelectedPercentage(`${selectedOperativePercentage.toString()}%`)
+      updatePercentages(
+        operativeIndex,
+        `${selectedOperativePercentage.toString()}%`
+      )
+      setSmv(
+        calculateSMV(`${selectedOperativePercentage.toString()}%`, totalSMV)
```
[the remark this answers is not in the document]
Comment (siapankina) [reply src/components/WorkOrders/SelectPercentage.js:119]: yes, you are right! I totally forgot about that

**Pull request merged**